docs/deployment.md

```diff
@@ -35,25 +35,35 @@ Options:
   --fd INTEGER                    Bind to socket from this file descriptor.
   --reload                        Enable auto-reload.
   --reload-dir TEXT               Set reload directories explicitly, instead
-                                  of using 'sys.path'.
-  --workers INTEGER               Number of worker processes. Not valid with
-                                  --reload.
-  --loop [auto|asyncio|uvloop]    Event loop implementation.  [default: auto]
+                                  of using the current working directory.
+  --workers INTEGER               Number of worker processes. Defaults to the
+                                  $WEB_CONCURRENCY environment variable if
+                                  available. Not valid with --reload.
+  --loop [auto|asyncio|uvloop|iocp]
+                                  Event loop implementation.  [default: auto]
   --http [auto|h11|httptools]     HTTP protocol implementation.  [default:
                                   auto]
-  --ws [none|auto|websockets|wsproto]
+  --ws [auto|none|websockets|wsproto]
                                   WebSocket protocol implementation.
                                   [default: auto]
   --lifespan [auto|on|off]        Lifespan implementation.  [default: auto]
-  --interface [auto|asgi3|agsi2|wsgi]
+  --interface [auto|asgi3|asgi2|wsgi]
                                   Select ASGI3, ASGI2, or WSGI as the
-                                  application interface.
-  --log-level [critical|error|warning|info|debug]
-                                  Log level.  [default: info]
-  --no-access-log                 Disable access log.
-  --proxy-headers                 Use X-Forwarded-Proto, X-Forwarded-For,
-                                  X-Forwarded-Port to populate remote address
-                                  info.
+                                  application interface.  [default: auto]
+  --env-file PATH                 Environment configuration file.
+  --log-config PATH               Logging configuration file.
+  --log-level [critical|error|warning|info|debug|trace]
+                                  Log level. [default: info]
+  --access-log / --no-access-log  Enable/Disable access log.
+  --use-colors / --use-colors     Enable/Disable colorized logging.
```

Comment:
Looks like this should be `--use-colors / --no-use-colors`

Reply:
ok I was sure I pasted it....now I'm sure